packages/bpk-component-phone-input/src/BpkPhoneInput.js

```diff
@@ -97,6 +101,20 @@ const BpkPhoneInput = (props: Props) => {
     disabled: !!disabled,
   };
 
+  let phoneDisplayValue;
+
+  if (countryCodeMask) {
+    if (value.startsWith('+')) {
+      if (value.startsWith(`+${dialingCode}`)) {
+        phoneDisplayValue = value;
+      } else {
+        phoneDisplayValue = `+${dialingCode} ${value.split(' ')[1]}`;
+      }
+    } else {
+      phoneDisplayValue = `+${dialingCode} ${value}`;
+    }
+  } else phoneDisplayValue = value;
```

Comment:
```suggestion
  } else {
    phoneDisplayValue = value;
}
```

This bit is still brace-less 😇

Reply:
whoa, you can give commit suggestions 😮 so cool! will accept this but will make another commit updating snapshots too

Reply:
done

Reply:
> whoa, you can give commit suggestions 😮

Haha yeah it only works on the open-source GitHub I think. It's a handy feature!